api/app/routers/hfi_calc.py

```diff
@@ -99,7 +98,31 @@ def get_prepared_request(
             planning_area_fire_starts=planning_area_fire_starts,
             planning_area_station_info=planning_area_station_info)
 
-    return result_request, request_loaded
+    return result_request, request_loaded, fire_centre_fire_start_ranges
+
+
+async def calculate_and_create_response(
+        session: Session,
+        result_request: HFIResultRequest,
+        fire_centre_fire_start_ranges: List[FireStartRange]) -> HFIResultResponse:
+    """ Calculate the HFI results and create a response object. """
+
+    # Calculate the HFI results.
```

Comment:
Minor: these comments are redundant with both the function comment and the function name